statsmodels/base/optimizer.py

```diff
@@ -343,6 +344,10 @@ def _fit_minimize(f, score, start_params, fargs, kwargs, disp=True,
 
     # Use bounds/constraints only if they're allowed by the method
     has_bounds = ['L-BFGS-B', 'TNC', 'SLSQP', 'trust-constr']
+    # Added in SP 1.5
+    has_bounds += ['Powell'] * (not SP_LT_15)
```

Comment:
I would prefer and explicit if

if not SP_LT_15:
     has_bounds += ...

that's quicker to understand